tools/cldr-code/src/main/java/org/unicode/cldr/util/VettingViewer.java

```diff
@@ -856,15 +866,36 @@ public int getLocaleCount(Organization org) {
         return localeCount;
     }
 
+    /**
+     * Get the list of locales to be summarized for the given organization
+     *
+     * @param org the organization
+     * @return the list of locale id strings
+     */
+    public ArrayList<String> getLocaleList(Organization org) {
+        final ArrayList<String> list = new ArrayList<>();
+        for (Level lv : Level.values()) {
+            final Map<String, String> sortedNames = getSortedNames(org, lv);
+            for (Map.Entry<String,String> entry : sortedNames.entrySet()) {
+                list.add(entry.getValue());
+            }
+        }
+        return list;
+    }
+
     public void generatePriorityItemsSummary(Appendable output, EnumSet<Choice> choices, T organization) {
         try {
             StringBuilder headerRow = new StringBuilder();
             headerRow
                 .append("<tr class='tvs-tr'>")
                 .append(TH_AND_STYLES)
+                .append("Level</th>")
+                .append(TH_AND_STYLES)
                 .append("Locale</th>")
                 .append(TH_AND_STYLES)
-                .append("Codes</th>");
+                .append("Codes</th>")
+                .append(TH_AND_STYLES)
+                .append("Progress</th>");
```

Comment:
all this html-generating code on the back end is techdebt, not improved in this pr